app/src/main/res/layout/app_version_fragment.xml

```diff
@@ -24,9 +24,9 @@
         android:layout_height="wrap_content"
         android:background="@drawable/selector_app_version"
         android:fontFamily="sans-serif"
-        android:paddingStart="16dp"
+        android:paddingStart="@dimen/app_version_padding_start"
```

Comment:
Rename this to `app_version_text_view_padding_start `

Reply:
Done